pageserver/src/tenant/timeline.rs

```diff
@@ -622,13 +622,78 @@ impl From<layer_manager::Shutdown> for GetVectoredError {
     }
 }
 
+pub enum ReadPathLayerId {
+    PersistentLayer(PersistentLayerKey),
+    InMemoryLayer(Range<Lsn>),
+}
```

Comment:
nit: can you add a comment saying that this enum is for observability purposes only and not used by the "real read path"?